src/pages/postgraphile/background_tasks.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: page
+path: /postgraphile/background-tasks/
+title: Background Tasks in PostGraphile 
+---
+
+In your GraphQL server typically most processing will be done in the "frontground", as part of the request-response cycle. HTTP is a blocking protocol: once a client sends a request, it will wait for a reponse. This blocking nature requires the server to be fast at serving responses, to avoid keeping the client waiting for ever.
```

Comment:
```suggestion
In your GraphQL server typically most processing will be done in the "foreground", as part of the request-response cycle. However, since clients typically wait for this request cycle to complete before allowing the user to continue to the next task, if this processing is slow it will make your application feel sluggish and can frustrate your users. Because of this desire for low-latency APIs, it's good practice to only perform essential calculations during the request-response cycle, and to "queue" additional tasks for background processing. 
```